hugegraph-core/src/main/java/org/apache/hugegraph/traversal/optimize/TraversalUtil.java

```diff
@@ -310,9 +311,9 @@ public static void extractAggregateFunc(Step<?, ?> newStep,
     }
 
     public static ConditionQuery fillConditionQuery(
-                                 ConditionQuery query,
-                                 List<HasContainer> hasContainers,
-                                 HugeGraph graph) {
+            ConditionQuery query,
+            List<HasContainer> hasContainers,
+            HugeGraph graph) {
```

Comment:
can we revert the style changes in all files.

Reply:
ok, I revert the style changes in all files

Reply:
> ok, I reverted the style changes in all files

after apply the `hugegraph-style.xml` we still get a string of differ foramt?

Reply:
> > ok, I reverted the style changes in all files
> 
> after apply the `hugegraph-style.xml` we still get a string of differ format?

The code after applying the `hugegraph-style.xml` is different from the origin format and these updates are automatically made by IDEA `save on action`.

Reply:
> > > ok, I reverted the style changes in all files
> > 
> > after apply the `hugegraph-style.xml` we still get a string of differ format?
> 
> The code after applying the `hugegraph-style.xml` is different from the origin format and these updates are automatically made by IDEA `save on action`.

get it，“save on action” need pay more attention，and I'll check it soon